infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -204,4 +205,17 @@ public void processStandbyChange(TopologyDescriptor newConfig) {
             }
         }
     }
+
+    /**
+     * Stop the current log replication event and start a full snapshot sync for all standby clusters
+     */
+    public void enforceSnapshotSync(DiscoveryServiceEvent event) {
+        for (CorfuLogReplicationRuntime standbyRuntime : runtimeToRemoteCluster.values()) {
```

Comment:
If runtimeToRemoteCluster is empty here, I think we won't have any log to see if we received the request or not...
It will be good to have a debug level log before this for loop to print eventId and runtimeToRemoteCluster size.